bool is_revolute
float64 damping
float64 friction
float64 torque_limit
float64 upper_limit
float64 lower_limit
float64 speed_limit
geometry_msgs/Vector3 axis
